### DISCUSSION TITLE
How to get metadata to sync to Trakt collections?

### DISCUSSION THREAD

**Discussion by alexkahler** [Taxel/PlexTraktSync#1624] [Support]
For some reason, my Plex metadata isn't included when syncing to the Trakt.tv Collections.

I'm referring to this: https://blog.trakt.tv/how-to-build-your-collection-146e6e80bcd7

I've seen the previous ticket opened at https://github.com/Taxel/PlexTraktSync/issues/117 but I couldn't seem to find the solution.

Please help! 🙏

**Comment by glensc** [comment]
I don't understand what are you asking. perhaps create issue instead?
